packages/nocodb/Dockerfile

```diff
@@ -70,25 +72,27 @@ ENV LITESTREAM_S3_SKIP_VERIFY=false \
     NODE_ENV=production \
     PORT=8080
 
-# Install additional runtime dependencies
-RUN apt-get update && apt-get install -y \
-    dumb-init \
+# Install runtime dependencies
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates `# required by wget for HTTPS support` \
     curl \
-    wget \
+    dumb-init \
+    wget `# for convenience of downstream users` \
     && curl -L "https://github.com/TomWright/dasel/releases/download/v2.8.1/dasel_linux_$(dpkg --print-architecture)" -o /usr/local/bin/dasel \
     && chmod +x /usr/local/bin/dasel \
+    && apt-get purge curl \
```

Comment:
Let's keep the curl

Reply:
ok, see https://github.com/nocodb/nocodb/pull/9756/commits/8644a21bec54c9f5626bc538215f95e36ba0af10